Formula/dnsviz.rb

```diff
@@ -44,8 +44,8 @@ def install
       system libexec/"bin/python", "setup.py", "build_ext", "--openssl=#{Formula["openssl"].opt_prefix}", "install"
     end
     venv.pip_install resources.reject { |r| r.name == "m2crypto" }
-    venv.pip_install_and_link buildpath
-    man1.install Dir["doc/man/*.1"]
+    system libexec/"bin/python", "setup.py", "build"
```

Comment:
In addition to the python modules, there are a number of files that go in `$prefix/share` `$prefix/share/doc`, etc.  The problem was that I could never get the data to install properly to those locations.  I had a discussion with @tdsmith on the (now deprecated) homebrew-discuss google group, and with his help, I finally settled on the `venv.pip_install_and_link buildpath`, but I realized since #7089 that I could just call `python setup.py build` and `python setup.py install` with `--prefix` (and the like) arguments and get the behavior I wanted.  It also matches the behavior of non-python packages, e.g., graphviz.  I know it's not as pretty, but it puts everything in the right place (and notice that I'm already using something similar for the `m2crypto` dependency, per the suggestion of @tdsmith ).